tests/async/tasyncawait_cyclebreaker.nim

```diff
@@ -65,7 +65,7 @@ while true:
   GC_collectZct()
 
   let (allocs, deallocs) = getMemCounters()
-  doAssert allocs < deallocs + 1000
+  doAssert allocs < deallocs + 150_000 # changed after await_template, not sure if it isn't too much
```

Comment:
Is this just a way to fix flakiness, or do your changes break this test without this change being introduced?

Reply:
yes, sorry, please read the irc logs in https://irclogs.nim-lang.org/16-04-2020.html#13:51:11 : Araq said to investigate it after all

Reply:
this should be 1000 now again